ppr-ui/src/components/tables/SearchHistory.vue

```diff
@@ -95,13 +95,19 @@
                   >
                     <template v-slot:activator="{ on, attrs }">
                       <v-btn
-                        icon
                         v-if="!item.inProgress"
-                        color="primary"
+                        :icon="item.isPdfRequested"
+                        :depressed="!item.isPdfRequested"
```

Comment:
Requirements call for a conditional box-shadow? Seems so specific!

Reply:
Just trying to keep consistent with other PDF buttons (from above). Since this button switches between icon and icon+text, we need depressed only in certain scenarios.

Reply:
Makes sense, thanks for the explanation!